packages/magellan-mocha-plugin/package.json

```diff
@@ -1,24 +1,27 @@
 {
-	"name": "testarmada-magellan-mocha-plugin",
-	"version": "9.0.1",
+	"name": "@automattic/testarmada-magellan-mocha-plugin",
+	"version": "10.0.0",
 	"description": "Magellan plugin to provide Mocha support",
 	"main": "index.js",
-	"scripts": {
-		"test": "mocha --slow 400 && npm run coverage && npm run check-coverage",
-		"lint": "eslint lib/** test/**",
-		"coverage": "istanbul cover _mocha",
-		"check-coverage": "istanbul check-coverage --statement 95 --function 95 --branch 90"
+	"author": "Automattic Inc.",
+	"publishConfig": {
+		"access": "public"
 	},
 	"repository": {
 		"type": "git",
-		"url": "git+https://github.com/TestArmada/magellan-mocha-plugin.git"
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/magellan-mocha-plugin"
 	},
-	"author": "",
-	"license": "MIT",
 	"bugs": {
-		"url": "https://github.com/TestArmada/magellan-mocha-plugin/issues"
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"license": "MIT",
+	"scripts": {
```

Comment:
Note these scripts have not been changed. In an upcoming PR I'll change them to fit with yarn workspaces and be consistent with the rest of the repo.

Reply:
As promised #51783